.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # dependencies
 /node_modules
+package-lock.json
```

Comment:
package.lock needed in gitignore?

Reply:
Yeah to avoid getting added in each merge commit, I have added this

Reply:
but if any package is updated or changes this will also be modified right. So you are suggesting no need of pushing that change when we take a pull just do npm i and keep the package.lock changes in local itself

Reply:
Yeah, package.json should be enough.. can be removed from gitignore.. either way it's fine